browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1741,20 +1741,14 @@ L.CanvasTileLayer = L.Layer.extend({
 	},
 
 	_onCalcFunctionUsageMsg: function (textMsg) {
-		var formulaAutocompletePopup = L.DomUtil.get('formulaautocompletePopup');
-		if (formulaAutocompletePopup)
-			this._map.fire('closeformulapopup');
+		this._map.fire('closeformulapopup');
 		this._map.fire('sendformulausagetext', {data: textMsg});
 	},
 
 	_onCalcFunctionListMsg: function (textMsg) {
 		if (textMsg.startsWith('hidetip')) {
-			var formulaAutocompletePopup = L.DomUtil.get('formulaautocompletePopup');
-            if (formulaAutocompletePopup)
-                this._map.fire('closeformulapopup');
-            var formulaUsagePopup = L.DomUtil.get('formulausagePopup');
-            if (formulaUsagePopup)
-                this._map.fire('closeformulausagepopup');
+            this._map.fire('closeformulapopup');
+            this._map.fire('closeformulausagepopup');
```

Comment:
Is there any good reason to have separate events for all these popups?
Can we maybe call just "close all popups" event? And then all popups will do just closePopup on that signal.
(we even have already such event somewhere and we can try to reuse that)

Reply:
closeformulausagepopup is not a popup, It is a dialog. I'll try your suggestion, if it works i'll use it